flink-table/flink-table-planner-blink/src/test/resources/org/apache/flink/table/planner/plan/batch/sql/DagOptimizationTest.xml

```diff
@@ -31,13 +31,11 @@ LogicalSink(name=[`default_catalog`.`default_database`.`sink2`], fields=[b, cnt]
          +- LogicalProject(b=[$1], a=[$0])
             +- LogicalUnion(all=[true])
                :- LogicalProject(a=[$0], b=[$1], c=[$2])
-               :  +- LogicalProject(a=[$0], b=[$1], c=[$2])
-               :     +- LogicalFilter(condition=[LIKE($2, _UTF-16LE'%hello%')])
-               :        +- LogicalTableScan(table=[[default_catalog, default_database, MyTable, source: [TestTableSource(a, b, c)]]])
+               :  +- LogicalFilter(condition=[LIKE($2, _UTF-16LE'%hello%')])
```

Comment:
Could you explain why this file is changed? I didn't expect any of your changes should effect this.